roles/confluent.common/tasks/remove_packages.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Get Package Facts
+  package_facts:
+
+- set_fact:
```

Comment:
one idea for the version changed is to inspect the version on `confluent-common` because all components have that package. And that package didn't have a name change like confluent-server/kafka/kafka-2.12 did